app/models/concerns/enc_output.rb

```diff
@@ -43,6 +43,7 @@ module EncOutput
   def to_enc
     own_attributes = {}
     self.attributes.each do |k, v|
+      v = send(k) if (v.is_a?(String) && v.start_with?(Encryptable::ENCRYPTION_PREFIX))
```

Comment:
I would prefer using more high-level methods than regexp matching and then calling generic `send`. What about something like:
```
v = decrypt_field(v) if self.is_a?(Encyrptable) && is_decryptable?(v)
```